internal/api/migrate/testdata/barchart_perses_dashboard.json

```diff
@@ -1,68 +1,68 @@
 {
-  "kind": "Dashboard",
-  "metadata": {
+    "kind": "Dashboard",
```

Comment:
For all the unit tests files, please tune your IDE to not alter the indentation already in place (which is 2 Spaces)